svc-api/handle/handlers.go

```diff
@@ -348,9 +347,9 @@ func GetMetadata(ctx iris.Context) {
 					models.Include{Namespace: "EthernetInterfaceCollection"},
 				},
 			},
-			models.Reference{URI: "http://redfish.dmtf.org/schemas/v1/EthernetInterfaces_v1.xml",
+			models.Reference{URI: "http://redfish.dmtf.org/schemas/v1/EthernetInterface_v1.xml",
 				TopInclude: []models.Include{
-					models.Include{Namespace: "EthernetInterfaces"},
+					models.Include{Namespace: "EthernetInterface"},
```

Comment:
Please add EthernetInterface version used in ODIMRA

Reply:
done